src/parser/parser.yy

```diff
@@ -1349,7 +1349,39 @@ yield_columns
     ;
 
 yield_column
-    : expression {
+    : KW_VERTEX {
+        $$ = nullptr;
+        throw nebula::GraphParser::syntax_error(@1, "please add alias when using vertex.");
+    }
+    | KW_VERTEX KW_AS name_label {
```

Comment:
The `vertex` is illegal, only `vertices` is ok.

Reply:
legal for : fetch prop on player "vid" YIELD vertex as a

Reply:
It's another thing. I think you should add the new rule to `subgraph` instead of in yield clause rule. yield clause is used by many other sentence instead of only `fetch` and `subgraph`.

Reply:
It is not necessary to have a separate yield for each statement (go, fetch edge, fetch vertex, subgraph, path, look up vertex, loop up edge),